sdk/quantum/quantum-jobs/package.json

```diff
@@ -1,116 +1,123 @@
 {
   "name": "@azure/quantum-jobs",
-  "author": "Microsoft Corporation",
-  "description": "Azure Quantum REST API client",
   "version": "1.0.0-beta.1",
-  "dependencies": {
-    "@azure/core-paging": "^1.1.1",
-    "@azure/core-http": "^1.2.0",
-    "tslib": "^2.0.0",
-    "@opentelemetry/api": "^0.10.2"
+  "description": "An isomorphic client library for Azure Quantum",
+  "sdk-type": "client",
+  "author": "Microsoft Corporation",
+  "license": "MIT",
+  "main": "dist/index.js",
+  "module": "dist-esm/src/index.js",
+  "types": "./types/latest/quantum-jobs.d.ts",
+  "engines": {
+    "node": ">=8.0.0"
   },
+  "scripts": {
+    "audit": "node ../../../common/scripts/rush-audit.js && rimraf node_modules package-lock.json && npm i --package-lock-only 2>&1 && npm audit",
+    "build:browser": "tsc -p . && cross-env ONLY_BROWSER=true rollup -c 2>&1",
+    "build:node": "tsc -p . && cross-env ONLY_NODE=true rollup -c 2>&1 && npm run extract-api",
+    "build": "npm run build:node && npm run build:browser",
+    "build:test:browser": "tsc -p . && cross-env ONLY_BROWSER=true rollup -c rollup.test.config.js 2>&1",
+    "build:test:node": "tsc -p . && cross-env ONLY_NODE=true rollup -c rollup.test.config.js 2>&1",
+    "build:test": "npm run build:test:node && npm run build:test:browser",
+    "build:samples": "echo Skipped.",
+    "check-format": "prettier --list-different --config ../../.prettierrc.json --ignore-path ../../.prettierignore \"src/**/*.ts\" \"test/**/*.ts\" \"*.{js,json}\"",
+    "clean": "rimraf dist dist-esm dist-browser test-dist test-browser types *.tgz *.log",
+    "coverage": "nyc --reporter=lcov --exclude-after-remap=false mocha -t 120000 dist-test/index.node.js --reporter ../../../common/tools/mocha-multi-reporter.js",
+    "execute:js-samples": "node ../../../common/scripts/run-samples.js samples/javascript/",
+    "execute:samples": "npm run build:samples && dev-tool samples run dist-samples/javascript dist-samples/typescript/dist/dist-samples/typescript/src/",
+    "extract-api": "tsc -p . && api-extractor run --local",
+    "format": "prettier --write --config ../../.prettierrc.json --ignore-path ../../.prettierignore \"src/**/*.ts\" \"test/**/*.ts\" \"*.{js,json}\"",
+    "integration-test:browser": "echo skipped",
```

Comment:
Browser tests are being skipped because we would need to launch the browser pop-up window for authentication and it couldn't be possible to do that with a headless browser.

/cc @sadasant